### PR TITLE
doc: add prerequistes checks to Astarte in 5 minutes

### PR DESCRIPTION
**Pull request by bettio**
Make sure users are using recent software versions.

### CODE REVIEW

File: doc/pages/tutorials/010-astarte_in_5_minutes.md [hunk 1]
```diff
@@ -6,10 +6,44 @@ This tutorial will guide you through bringing up your Astarte instance, creating
...
+astartectl 0.11.x is recommended:
+```sh
+$ astartectl version
+astartectl 0.11.0
+```
+
+We tested this procedure using Ubuntu 18.04 and macOS 10.15 Catalina, but any other recent
+operating system is fine.
+Linux distribution version can be checked using following command:
+```sh
```
Comment (drf) [inline doc/pages/tutorials/010-astarte_in_5_minutes.md:36]: add a newline before the code block

File: doc/pages/tutorials/010-astarte_in_5_minutes.md [hunk 2]
```diff
@@ -6,10 +6,44 @@ This tutorial will guide you through bringing up your Astarte instance, creating
...
+docker compose version 1.17.1, build unknown
+```
+
+astartectl 0.11.x is recommended:
+```sh
+$ astartectl version
+astartectl 0.11.0
+```
+
+We tested this procedure using Ubuntu 18.04 and macOS 10.15 Catalina, but any other recent
```
Comment (drf) [inline doc/pages/tutorials/010-astarte_in_5_minutes.md:33]: Remove the "we" when writing documentation.

`This procedure has been tested on several systems, and is validated and maintained against Ubuntu 18.04 and macOS 10.15 Catalina, but any other modern operating system should work.

File: doc/pages/tutorials/010-astarte_in_5_minutes.md [hunk 3]
```diff
@@ -6,10 +6,44 @@ This tutorial will guide you through bringing up your Astarte instance, creating
...
+Linux distribution version can be checked using following command:
+```sh
+$ lsb_release --all
+Distributor ID: Ubuntu
+Description:    Ubuntu 18.04.4 LTS
+Release:        18.04
+Codename:       bionic
+```
+
+If any software component is older than the recommended you might succeed with this tutorial,
```
Comment (drf) [inline doc/pages/tutorials/010-astarte_in_5_minutes.md:44]: Remove this, it's redundant and confusing

File: doc/pages/tutorials/010-astarte_in_5_minutes.md [hunk 4]
```diff
@@ -6,10 +6,45 @@ This tutorial will guide you through bringing up your Astarte instance, creating
...
+astartectl 0.11.x is recommended:
+
+```sh
+$ astartectl version
+astartectl 0.11.0
+```
+
+This procedure has been tested on several systems, and is validated and maintained against
+Ubuntu 18.04 and macOS 10.15 Catalina, but any other modern operating system should work.
+Linux distribution version can be checked using following command:
```
Comment (drf) [inline doc/pages/tutorials/010-astarte_in_5_minutes.md:38]: Remove the Linux distribution part entirely

File: doc/pages/tutorials/010-astarte_in_5_minutes.md [hunk 5]
```diff
@@ -6,10 +6,45 @@ This tutorial will guide you through bringing up your Astarte instance, creating
 
 First of all, please keep in mind that **this setup is not meant to be used in production**: by default, no persistence is involved, the installation does not have any recovery mechanism, and you will have to restart services manually in case something goes awry. This guide is great if you want to take Astarte for a spin, or if you want to use an isolated instance for development.
 
-You will need a machine with at least 4GB of RAM, with [Docker](https://www.docker.com/) and [astartectl](https://github.com/astarte-platform/astartectl) installed. If you don't have `astartectl` installed on your machine yet, you should install it by following the instructions in [astartectl's README](https://github.com/astarte-platform/astartectl#installation)
+You will need a machine with at least 4GB of RAM, a recent 64-bit operating system with [Docker](https://www.docker.com/) and [astartectl](https://github.com/astarte-platform/astartectl) installed. If you don't have `astartectl` installed on your machine yet, you should install it by following the instructions in [astartectl's README](https://github.com/astarte-platform/astartectl#installation)
```
Comment (rbino) [inline doc/pages/tutorials/010-astarte_in_5_minutes.md:9]: While we're at it, mention `docker-compose` explicitly (it's a separate package from Docker in, e.g., Ubuntu)

**Pull request merged**